src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax.model;
 
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity(name = "messages")
 public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+
+    @OneToMany(cascade = {CascadeType.ALL},
```

Comment:
I think it will be enough: CascadeType.PERSIST, CascadeType.DELETE

Reply:
if I use CascadeType.PERSIST, CascadeType.REMOVE two test dont work